cocos2d/core/base-nodes/CCAtlasNodeCanvasRenderCmd.js

```diff
@@ -30,8 +30,7 @@
         cc.Node.CanvasRenderCmd.call(this, renderableObject);
         this._needDraw = false;
         this._colorUnmodified = cc.color.WHITE;
-        this._originalTexture = null;
-        this._texture = null;
+        this._renderTexture = null;
```

Comment:
It's very confusing with our current RenderTexture, we'd better rename it to `_textureToRender` etc